common/formatting/align.cc

```diff
@@ -323,14 +325,42 @@ static void ComputeCellWidths(AlignmentMatrix* matrix) {
 typedef std::vector<AlignedColumnConfiguration> AlignedFormattingColumnSchema;
 
 static AlignedFormattingColumnSchema ComputeColumnWidths(
-    const AlignmentMatrix& matrix) {
+    const AlignmentMatrix& matrix,
+    const std::vector<AlignmentColumnProperties> column_properties) {
```

Comment:
make sure to pass vectors by reference so that they are not copied.

Reply:
Fixed